pkg/envtest/envtest_test.go

```diff
@@ -58,10 +58,28 @@ var _ = Describe("Test", func() {
 	})
 
 	Describe("InstallCRDs", func() {
+
+		It("should set ErrorIfCRDPathMissing through env settings", func(done Done) {
```

Comment:
it's unclear to me what this is testing, since it doesn't seem to be testing that the setting gets automatically propogated from `Environment` to `InstallCRDs`

Reply:
Yes, it only validates the new option. This was to keep the other tests unchanged here.
https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/envtest/envtest_test.go#L155 

My other thought was to make this change directory in [BeforeSuite](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/envtest/envtest_suite_test.go#L37) and update all the tests accordingly.

```
var _ = BeforeSuite(func(done Done) {
	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
	env = &Environment{
++   	    ErrorIfCRDPathMissing: true,
        }
	_, err := env.Start()
```
Any opposition to this?